src/components/Hero/Labels.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { typography, spacing } from '@/theme';
+
+const Labels = ({ labels }) => {
+  return (
+    <View style={styles.container}>
+      {labels.map((label, idx) => (
+        <View style={styles.row} key={idx}>
+          {idx > 0 && <Text style={styles.divider}>{'\u2B24'}</Text>}
+          <Text style={[styles.label, typography.text]}>{label.name}</Text>
+        </View>
```

Comment:
In general, we use flatList to render big collections. I am not sure how many labels a movie has, but take this into considerations for future collections